libcloudforensics/providers/gcp/internal/common.py

```diff
@@ -266,12 +267,10 @@ def ExecuteRequest(client: 'googleapiclient.discovery.Resource',
       request = getattr(client, func)
       response = request(**kwargs).execute()
     except (RefreshError, DefaultCredentialsError) as exception:
-      error_msg = (
-          '{0:s}\n'
-          'Something is wrong with your Application Default '
-          'Credentials. Try running: '
-          '$ gcloud auth application-default login'.format(str(exception)))
-      raise RuntimeError(error_msg)
+      raise errors.CredentialsConfigurationError(
+          'Something is wrong with your Application Default Credentials. Try '
+          'running: $ gcloud auth application-default login: {0!s}'.format(
```

Comment:
I'd have the exception message before the gclout auth command, I believe it'd be easier to read

Reply:
I think we have this kind of multiple imports a bit everywhere. I opt to leave it as is for now and opened